cropper/src/main/java/com/canhub/cropper/CropImageContractOptions.kt

```diff
@@ -49,6 +49,30 @@ data class CropImageContractOptions @JvmOverloads constructor(
         cropImageOptions.cropShape = cropShape
         return this
     }
+    /**
+     * To set the shape of the cropper corner (RECTANGLE / OVAL)
+     * Default: RECTANGLE
+     */
+    fun setCropCornerShape(cornerShape: CropImageView.CropCornerShape): CropImageContractOptions {
+         cropImageOptions.cornerShape = cornerShape
+        return this
+    }
+
+    /**
+     * To set the fill color of the Oval crop corner
+     */
+    fun setCircleCornerFillColor(circleFillColor:Int):CropImageContractOptions{
```

Comment:
Do we expect the value on RGB or Hex or Android Int? Would be nice to have some tip on the variable name, like `circleFillColorHexValue` and to have it clear on the documentation